engine/dlib/src/remotery/lib/Remotery.c

```diff
@@ -2700,6 +2708,16 @@ static rmtError rmtHashTable_Insert(rmtHashTable* table, rmtU32 key, rmtU64 valu
     rmtU32 index_mask = table->maxNbSlots - 1;
     rmtU32 index = key & index_mask;
 
+// DEFOLD
+    if (key == 0) {
+        if (g_EmptyHash == 0) {
+            g_EmptyHash = dmHashString32(g_EmptyString);
+        }
+        key = g_EmptyHash;
+        dmLogError("REMOTERY: DEFOLD: rmtHashTable_Insert: The hash was 0x%08x. Setting it to 0x%08x ('%s')", key, g_EmptyHash, g_EmptyString);
+    }
+// END DEFOLD
```

Comment:
Before the assert, we check if it's 0, and log it.
Then we change it to the "empty" hash.